src/metabase/models/user.clj

```diff
@@ -188,13 +188,23 @@
            common-name)
       (assoc :common_name common-name))))
 
+(defn- maybe-get-name-from-api-key
+  "If the user is an API Key user, replace any existing name with the name from the API Key table."
+  [{:keys [type] :as user}]
+  (cond-> user
+    (= :api-key type)
+    (assoc :first_name (i18n/tru "API Key:")
```

Comment:
nit: our convention is usually to refer translation macros. `tru` is already available in this namespace without the `i18n` alias
```suggestion
    (assoc :first_name (tru "API Key:")
```

Reply:
Still need to be updated?